<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListView" name="tblShowResults">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="lblSetFirstInput">
            <property name="text">
             <string>Set Speed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblSetSecondInput">
            <property name="text">
             <string>Set Noise</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblJessPath">
            <property name="text">
             <string>Jess Path:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QSpinBox" name="spnSetFirstInput">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>129</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnSetSecondInput">
            <property name="minimum">
             <number>26</number>
            </property>
            <property name="maximum">
             <number>74</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lnlJessPath"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btnSaveInputsValues">
          <property name="text">
           <string>Save Inputs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRunJessProcess">
          <property name="text">
           <string>Run Jess</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSetJessPath">
          <property name="text">
           <string>Set Jess Path</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>472</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuProgram"/>
  </widget>
  <action name="actAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
